src/replicache.ts

```diff
@@ -149,16 +201,27 @@ export default class Replicache implements ReadTransaction {
     window.addEventListener('storage', this._onStorage);
   }
 
+  /**
+   * Whether Replicache thinks that we are online. This is a heuristic and we
+   * treat failure to sync as being offline.
```

Comment:
```suggestion
   * A rough heuristic for whether the client is currently online. Note that there is no way to know
   * for certain whether a client is online - the next request can always fail. This is true if the last
   * sync attempt succeeded, and false otherwise.
```